### ISSUE TITLE
AdFind.zip from payloads requires a password

### ISSUE THREAD

**Issue by arty-hlr** [mitre/emu#41]
**Describe the bug**
AdFind.zip that is downloaded for the emu plugin now requires a password

**To Reproduce**
1. ./download_payloads.sh

**Expected behavior**
All payloads are downloaded and unzipped.

**Screenshots**
See [here](http://www.joeware.net/freetools/tools/adfind/) at the bottom in red, the zip is now protected with a password that is included in the zip itself (`NotMalware`)
![image](https://github.com/mitre/emu/assets/38589602/ef0a33f2-bd0a-4260-b92e-7c2ac55ebe17)

**Desktop:**
 - OS: Kali
 - Browser: Firefox
 - Version: Caldera 4.2.0

**Comment by arty-hlr** [comment]
Yes. Still that doesn't work for a script and that password should just be included so that AdFind.exe can be extracted.

**Comment by fstarnaud** [comment]
The password could be supplied on the command-line using the `-P` argument:

`unzip -P NotMalware payloads/AdFind.zip -d payloads/`

**Comment by arty-hlr** [comment]
The issue is that I had to look for the original AdFind webpage to find out that there's a password now. This is not reflected in the `./download_payloads.sh` script, and it should be changed to unzip with that password directly.

**Comment by fstarnaud** [comment]
Ah! Just noticed there are already PRs  to fix this...

**Comment by elegantmoose** [comment]
Closed with #45.

Sorry for tardiness everyone.\
